frontend-vue-js/src/plugins/router.js

```diff
@@ -67,6 +68,11 @@ const routes = [
                 component: PharmacistLanding,
                 name: 'pharmacist',
                 path: ''
+            },
+            {
+                component: PharmacistProfile,
+                name: 'profile',
+                path: '/pharmacist/profile'
```

Comment:
This is a child component, the parent's path is already '/pharmacist'. You should define the child's path as just 'profile' (with no slash) to get this path.

Reply:
Fixed